app/src/main/java/com/teamagam/gimelgimel/app/view/fragments/ViewerFragment.java

```diff
@@ -376,4 +417,66 @@ private void onCreateGeographicMessage(PointGeometry pointGeometry) {
         // TODO: Update argument type and name
         void onFragmentInteraction(Uri uri);
     }
+
+    private static class ElapsedTimeUserLocationSymbolizer implements UsersLocationViewModel.UserLocationSymbolizer {
+
+        private static final Logger sLogger = LoggerFactory.create(
+                ElapsedTimeUserLocationSymbolizer.class);
+
+        //TODO: config , add these to the static configuration file after merging configuration feature branch
+        private static final long USER_LOCATION_STALE_THRESHOLD_MS = 60 * 1000;
```

Comment:
do you think that we need to remove (transparent) users with bigger threshold for